attributes/default.rb

```diff
@@ -110,3 +110,5 @@
 default['nginx']['ohai_plugin_enabled'] = true
 
 default['nginx']['load_modules'] = []
+
+default['nginx']['pcre_jit'] = 'off'
```

Comment:
Would this not be better as a true false?

Reply:
There are other parameters that are using the `on`/`off` values, but well I've updated my code to follow your suggestion.